src/api/apps/articles/model/save.coffee

```diff
@@ -187,8 +190,14 @@ sanitize = (article) ->
   sanitized
 
 sanitizeLink = (urlString) ->
+  return unless urlString
   u = url.parse urlString
-  if u.protocol then urlString else 'http://' + u.href
+  return urlString if u.protocol is "https:"
+
+  if u.protocol
+    urlString.replace("http:", "https:") if u.hostname is "artsy.net"
```

Comment:
My coffeescript may be rusty, but does this line risk returning `null` when there _is_ a protocol but the hostname isn't artsy.net?

Reply:
Also, a hostname of `www.artsy.net` _should_ trigger this treatment. (Ideally even all `*.artsy.net` hostnames, but I know there are one or 2 that were never upgraded for SSL.)

Reply:
Sorry if this^ wasn't clear. The most recent commit strips `www` out of links, but it's preferable to keep that.

Basically, all of `http://artsy.net/*`, `http://www.artsy.net/*`, `artsy.net/*` and `www.artsy.net/*` should be replaced by `https://www.artsy.net/*`.